### DISCUSSION TITLE
New versions not working on old Mac

### DISCUSSION THREAD

**Discussion by johnsteed3** [rustdesk/rustdesk#9674] [Q&A]
Hello,

i would like to give you a few feed back about my experience. I use several mac, iMac 27 2009, Macbook Pro 2009, iMac 21,5 2011, iMac 2017... 

With the last version of Rustdesk, only the 2017 models works. All the models from 2009 to 2011 require the Rustdesk 1.9. When I try a newer version (1.2.0 to 1.3.1) I can start the app, the system asks me if I want to start the app and then nothing more happens. 

All the 2009 to 2011 Mac are running under 10.15.7 Catalina patched with OpenCore.
So, I guess something is specific with the old Mac or OpenCore but I don't know how to fix the problem.

If someone knows how to run the last version of Rustdesk on the old iMac (2009 to 2011) I'll appreciate. 

Kind regards.
John

**Comment by xlionjuan** [comment]
You should format and install Linux.
